<?xml version="1.0" encoding="UTF-8" ?>
<!--
  ~ Copyright 2000-2011 JetBrains s.r.o.
  ~
  ~ Licensed under the Apache License, Version 2.0 (the "License");
  ~ you may not use this file except in compliance with the License.
  ~ You may obtain a copy of the License at
  ~
  ~ http://www.apache.org/licenses/LICENSE-2.0
  ~
  ~ Unless required by applicable law or agreed to in writing, software
  ~ distributed under the License is distributed on an "AS IS" BASIS,
  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  ~ See the License for the specific language governing permissions and
  ~ limitations under the License.
  -->

<testsuite errors="0" failures="0" hostname="ruspd-student3" name="TestCase" tests="2" time="0.062"
           timestamp="2008-10-30T17:11:25">
  <properties/>
  <testcase classname="TestCase" name="test1" time="0.031">
    <failure message="Assertion message form test" type="junit.framework.AssertionFailedError">
      junit.framework.AssertionFailedError: Assertion message form test
      at TestCase.test(Unknown Source)
    </failure>
  </testcase>
  <testcase classname="TestCase" name="test2" time="0.031">
    <error message="Error message from test" type="java.lang.NullPointerException">java.lang.NullPointerException:
      Error message from test
      at TestCase.test(Unknown Source)
    </error>
  </testcase>